whylog/tests/tests_front/test_unit_whylog_front.py

```diff
@@ -0,0 +1,38 @@
+from unittest import TestCase
+
+import six
+from mock import MagicMock, patch
+from testconfig import config
+
+from whylog.front import Front
+from whylog.front.utils import FrontInput, LocallyAccessibleLogOpener
+
+
+def get_test_front_input():
+    log_opener = LocallyAccessibleLogOpener(path=six.moves.cStringIO)
+    return FrontInput(offset=0, line_content='effect', resource_location=log_opener)
+
+
+class TestUnitWhylogFront(TestCase):
+    def setUp(self):
+        self.front = Front(config=config)
+
+    def test_get_causes(self):
+        front_input = get_test_front_input()
+
+        investigation_result = MagicMock()
+        with patch('whylog.client.Client.get_causes') as mock:
+            mock.return_value = investigation_result
+            causes = self.front.get_causes(front_input)
+
+        assert causes is investigation_result
```

Comment:
This 6-line test checks a one-line trivial method of Front (`return self.client.get_causes(input_)`). It doesn't bring much value, but it brings test fragility.